libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/ShareGranteeBase.tsx

```diff
@@ -54,6 +65,20 @@ export const ShareGranteeBase: React.FC<IShareGranteeBaseProps> = (props) => {
                 onDelete={onGranteeDelete}
             />
             <ContentDivider />
+            <SharedObjectCheckboxControl
```

Comment:
Could you separate SharedObjectCheckboxControl and SharedObjectCheckboxControl into component

Reply:
Fixed.